services/slam/builtin/orbslam_yaml.go

```diff
@@ -85,7 +85,7 @@ func (slamSvc *builtIn) orbCamMaker(camProperties *transform.PinholeCameraModel)
 	if orbslam.Stereob, err = slamSvc.orbConfigToFloat("stereo_b", 0.0745); err != nil {
 		return nil, err
 	}
-	tmp, err := slamSvc.orbConfigToInt("rgb_flag", 1)
+	tmp, err := slamSvc.orbConfigToInt("rgb_flag", 0)
```

Comment:
because we use the [IM_COLOR](https://docs.opencv.org/3.4/d8/d6a/group__imgcodecs__flags.html#gga61d9b0126a3e57d9277ac48327799c80af660544735200cbe942eea09232eb822) flag for our color images, we want to ensure orbslam knows our images will be BGR

Reply:
This was also in https://viam.atlassian.net/browse/DATA-535, so I'll skip the change there.